src/services/jsonforms/index.ts

```diff
@@ -80,6 +98,15 @@ const isMultilineText = (schema: JsonSchema): boolean => {
     }
 };
 
+const isDateText = (schema: JsonSchema): boolean => {
+    if (schema.type === 'string' && Object.hasOwn(schema, 'format')) {
+        // eslint-disable-next-line @typescript-eslint/dot-notation
+        return schema['format'] === 'date';
```

Comment:
Any reason not to leverage the `Formats` enumeration in place of the `'date'` (here) and `'time'` (below) strings?

Reply:
Considered it but wanted to follow how it was done before.